cmd/ocitools/validate.go

```diff
@@ -98,531 +54,15 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
```

Comment:
Alternatively, use:

    func NewValidator(spec *rspec.Spec, bundlePath string, hostSpecific bool) Validator

and load from `{bundlePath}/config.json` if `spec` is `nil`.